src/components/index/StartExploring.tsx

```diff
@@ -0,0 +1,17 @@
+import { IoChevronDown } from "react-icons/io5";
+import { Container } from "@components/commons/Container";
+
+export function StartExploring() {
+  return (
+    <Container className="flex justify-center md:mt-0 mt-9 hidden md:flex">
```

Comment:
`hidden` has the same CSS properties as `flex`

Reply:
updated